GameBoard/appsettings.json

```diff
@@ -1,6 +1,8 @@
 {
-  "ConnectionStrings": {  
-    "DefaultConnection": "Host=localhost;Port=5432;Database=gameboard_db;UserId=postgres;Password=postgres"
+  "ConnectionStrings": {
+    "DefaultConnection": "Host=localhost;Port=5432;Database=gameboard_db;UserId=postgres;Password=postgres",
```

Comment:
This connection string is invalid now, it should be removed.

Reply:
Now it should work.